src/data/showcases.js

```diff
@@ -1216,6 +1216,15 @@ export const Showcases = [
     source: null,
     tags: ["dex", "token"],
   },
+  {
+    title: "Liqwid",
+    description:
+      "Liqwid is a non-custodial pooled lending protocol with liquid staking built on Cardano.",
+    preview: require("./showcase/liqwid.png"),
+    website: "https://app.liqwid.finance/",
```

Comment:
```suggestion
    website: "https://liqwid.finance/",
```
The project home page has an introduction for the app, links to documentation & support, people behind the project, etc... and a big button at the top that says **Go to the app** in case that is all people really want. 😉

Reply:
Good point. I debated which link to include, but I agree that directing new users to the main site makes the most sense. 😃